src/corelibs/U2Core/src/util/TextUtils.cpp

```diff
@@ -214,4 +214,16 @@ QStringList TextUtils::split(const QString &text, int chunkSize) {
     return result;
 }
 
+bool TextUtils::checkFirstListContainsSecond(const QStringList &firstList, const QStringList &secondList) {
+    if (firstList.size() < secondList.size()) {
+        return false;
+    }
+    for (const QString &valueFromSecondList : secondList) {
+        if (!firstList.contains(valueFromSecondList)) {
+            return false;
+        }
+    }
+    return true;
```

Comment:
You can use ```firstList.toSet().contains(secondList.toSet())```
But you can leave this method, if you like.

Reply:
I removed this method completely and made GUITest use QSet<QString> for labels.